### PR TITLE
terraform-inventory: init at 0.7-pre

### PR DESCRIPTION
**Pull request by htr**
###### Motivation for this change
adds terraform-inventory

###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [x] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/networking/cluster/terraform-inventory/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    sha256 = "0wwyi2nfyn3wfpmvj8aabn0cjba0lpr5nw3rgd6qdywy7sc3rmb1";
+  };
+
+  goDeps = ./deps.nix;
+
+  meta = {
+    homepage = https://github.com/adammck/terraform-inventory;
+    description = "Terraform state to ansible inventory adapter";
+    platforms = stdenv.lib.platforms.all;
+    license = stdenv.lib.licenses.mit;
```
Comment (Mic92) [inline pkgs/applications/networking/cluster/terraform-inventory/default.nix]: Do you want to maintain it?

Comment (htr) [inline pkgs/applications/networking/cluster/terraform-inventory/default.nix]: yes, I can do that. Should I add myself to `lib/maintainers.nix` ?

Comment (Mic92) [inline pkgs/applications/networking/cluster/terraform-inventory/default.nix]: yes.

**Comment by htr** [comment]
updated

**Comment by Mic92** [comment]
Thanks!

**Pull request merged**